robotiq_description/urdf/robotiq_sim_gripper.ros2_control.xacro

```diff
@@ -3,6 +3,7 @@
 
     <xacro:macro name="robotiq_sim_gripper_ros2_control" params="
       name
+      sim_ignition:=false
       sim_isaac:=false
       use_fake_hardware:=false">
```

Comment:
Should one of these default to `true`?

Reply:
fake hardware?

Reply:
Ya, that seems to make the most sense to me.